pyas2lib/cms.py

```diff
@@ -163,7 +171,11 @@ def encrypt_message(data_to_encrypt, enc_alg, encryption_cert):
                                     }
                                 ),
                                 "key_encryption_algorithm": cms.KeyEncryptionAlgorithm(
-                                    {"algorithm": cms.KeyEncryptionAlgorithmId("rsa")}
+                                    {
+                                        "algorithm": cms.KeyEncryptionAlgorithmId(
```

Comment:
this is a big change right, before we were sending rsa as the algorithm id and now its rsaes_pkcs1v15?

Reply:
Actually not, "rsa" and "rsaes_pkcs1v15" are resolved to the same value, so when sending "rsaes_pkcs1v15" which is the current default, then it will be the same value as if we sent "rsa". 

https://github.com/wbond/asn1crypto/blob/b763a757bb2bef2ab63620611ddd8006d5e9e4a2/asn1crypto/cms.py#L676-L686